package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-gauge",
-  "version": "3.1.3",
+  "version": "4.0.0",
```

Comment:
Can you also change this back to `"version": "3.1.3",` as it gets bumped automatically on release.

Reply:
Done!